pkg/resources/cluster.go

```diff
@@ -53,3 +55,27 @@ func getCurrentVersion(versionHistory []configv1.UpdateHistory, log l.Logger) (f
 
 	return version, nil
 }
+
+func GetClusterInfrastructure(ctx context.Context, c client.Client) (*configv1.Infrastructure, error) {
+	infra := &configv1.Infrastructure{}
+	if err := c.Get(ctx, types.NamespacedName{Name: "cluster"}, infra); err != nil {
+		return nil, fmt.Errorf("failed to retrieve cluster infrastructure: %w", err)
+	}
+	return infra, nil
+}
+
+func GetClusterType(infra *configv1.Infrastructure) (string, error) {
+
+	switch infra.Status.PlatformStatus.Type {
+	case configv1.AWSPlatformType:
+		for _, tag := range infra.Status.PlatformStatus.AWS.ResourceTags {
+			if tag.Key == "red-hat-clustertype" {
```

Comment:
Pity the format is different, we might have gotten away with using the key as the label key.

Reply:
It could be possible. Our plan for the initial dashboards was to only expose the bear minimum needed. What is the strong agreement for including the value?

Reply:
It would give us metrics on manged vs ccs.

Reply:
While I agree with having that metric. The question becomes where does it stop in the initial implementation. The metric for quota value, multi AZ support and OSD version could all be exported. The question becomes how usefully is the information. By exporting only what is need the cardinality of the metrics will be kept low and lower is better. Cardinality will not be an issue for us any time soon it is still good practice to try keep it as low as possible.

Reply:
Design document to be updated with affected parties to comment and agree